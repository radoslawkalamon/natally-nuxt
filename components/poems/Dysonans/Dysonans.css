.dysonans {
  display: flex;
  flex-flow: column nowrap;
  gap: 3em;
  text-align: center;
}

.dysonans__superscript {
  position: relative;
  top: -20px;
}

.dysonans__subscript {
  position: relative;
  top: 20px;
}
