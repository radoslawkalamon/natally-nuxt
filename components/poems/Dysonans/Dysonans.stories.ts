import type { Meta, Story } from '@storybook/vue'
import PoemsDysonans from '@/components/poems/Dysonans/Dysonans.vue'
import WrappersTextPoems from '@/components/wrappers/TextPoems/TextPoems.vue'

const meta: Meta = {
  title: 'Poems / Dysonans',
  component: PoemsDysonans
}
export default meta

export const Default: Story = () => ({
  components: {
    PoemsDysonans,
    WrappersTextPoems
  },
  template: `<div class="__storybook-section__">
    <WrappersTextPoems>
      <PoemsDysonans />
    </WrappersTextPoems>
  </div>`
})
