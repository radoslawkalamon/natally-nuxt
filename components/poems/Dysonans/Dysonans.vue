<template>
  <div
    class="dysonans"
    data-test="poems-dysonans"
  >
    <p>
      Czasami <span class="dysonans__superscript">sztuka wysoka</span>
    </p>
    <p>
      to
    </p>
    <p>
      kompletne <span class="dysonans__subscript">dno.</span>
    </p>
  </div>
</template>

<script src="./Dysonans.ts" lang="ts"></script>
<style src="./Dysonans.css" scoped></style>
