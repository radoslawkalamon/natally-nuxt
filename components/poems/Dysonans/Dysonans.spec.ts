import Dysonans from '@/components/poems/Dysonans/Dysonans.vue'
import { shallPassUnitSanityTest } from '@/devtools/jest.common.spec'

describe('Poems / Dysonans', () => {
  describe('Unit', () => {
    shallPassUnitSanityTest({ component: Dysonans })
  })
})
