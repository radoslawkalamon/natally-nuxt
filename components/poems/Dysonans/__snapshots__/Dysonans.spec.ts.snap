// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poems / Dysonans Unit shall pass unit sanity test 1`] = `
"<div data-test=\\"poems-dysonans\\" class=\\"dysonans\\">
  <p>
    Czasami <span class=\\"dysonans__superscript\\">sztuka wysoka</span></p>
  <p>
    to
  </p>
  <p>
    kompletne <span class=\\"dysonans__subscript\\">dno.</span></p>
</div>"
`;
